src/app.js

```diff
@@ -7,23 +7,29 @@ import { serve, setup } from 'swagger-ui-express';
 import morgan from 'morgan';
 import sgMail from '@sendgrid/mail';
 import cors from 'cors';
-import formData from 'express-form-data';
+// import formData from 'express-form-data';
 // eslint-disable-next-line import/no-named-as-default-member
+import http from 'http';
 import routes from './routes/index';
+import { io } from './utils/socket.utils';
 
 const app = express();
-app.use(cors());
+const server = http.createServer(app);
+io.attach(server);
+app.use(
+  cors({
+    origin: '*'
+  })
+);
 app.use(morgan('dev'));
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
-app.use(formData.parse());
-app.use(formData.union());
+// app.use(formData.parse());
```

Comment:
can you un comment these lines and the line 10

Reply:
done